pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -120,42 +121,30 @@ public ViewFilter(String filterName, Map<String, List<Object>> columns, String i
         this.realmId = realmId;
     }
 
-    private static SimpleResult checkUniqueFilterName(String suggestedName) {
-        SimpleResult results = inTransaction((conn) -> {
-            SimpleResult dbVals = new SimpleResult();
+    /**
+     * Returns true if a filter with the given name exists in the database,
+     * false if it does not exist.
+     */
+    public static boolean doesFilterExist(String filterName) {
```

Comment:
Does it make sense to add a test for this method, or is manual testing sufficient?

Reply:
`saved-views-custom-columns.spec.ts` and `saved-views-custom-columns-filter-condition.spec.ts` do cover this via PW, so we'll catch any issues there.  I think if we did add another test, it would be most helpful to test `processRequest` here.  Since we're time boxing to a day, I'd lean towards merging this as-is, seeing what comes out of PW, and if there are subsequent issues, I can put in a more detailed junit test.

Reply:
Do we have a way to test `processRequest`?
I've tried to deal with the Spark Java classes on a couple of occasions, including a few hours today, and have found it very frustrating.